README.md

```diff
@@ -25,18 +25,30 @@ To use the log viewer, choose one of the UI implementation, one of the logging f
 //build.gradle
 
 // Choose one of those UI implementation:
-implementation "io.github.qupath:logviewer-ui-main:0.1.0"
-implementation "io.github.qupath:logviewer-ui-textarea:0.1.0"
-implementation "io.github.qupath:logviewer-ui-richtextfx:0.1.0"
+implementation "io.github.qupath:logviewer-ui-main:0.1.0-SNAPSHOT"
```

Comment:
It'll be easier to update this if the version is extract as a variable

Something like
```groovy
logViewerVersion = "0.1.0-SNAPSHOT"

implementation "io.github.qupath:logviewer-ui-main:$logViewerVersion"
implementation "io.github.qupath:logviewer-ui-textarea:$logViewerVersion"
implementation "io.github.qupath:logviewer-ui-richtextfx:$logViewerVersion"
```
(although need to check if the variable is in an acceptable place, and doesn't require `ext { ... }` because Gradle is fussy about that...)